### DISCUSSION TITLE
not installing in laravel ^10.10

### DISCUSSION THREAD

**Discussion by mehrang13** [spatie/laravel-medialibrary#3337] [General]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires spatie/laravel-medialibrary 9.0.0 -> satisfiable by spatie/laravel-medialibrary[9.0.0].
    - spatie/laravel-medialibrary 9.0.0 requires illuminate/bus ^7.0|^8.0 -> found illuminate/bus[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by MattWiseParking** [comment]
its because your updating the package without its dependencies.  try using the update with dependencies flag when using composer update

**Comment by parallels999** [comment]
> Root composer.json requires spatie/laravel-medialibrary 9.0.0

`laravel:^10.10` needs `laravel-medialibrary:^10.0`
